src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area, Drawable, Randomizable {
```

Comment:
The class name 'Figure' should not be a verb. However, in this case, 'Figure' is a noun, which is appropriate for a class name, so no change is needed here.